int32  userID   # The user who sent the audio command  
int32  robotID  # -1 all, 0 none, 107 specific
string txt      # The text of the user's audio sent to audio_command.msg
